<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright (C) 2018  Giuseppe Massimo Bertani gmbertani(a)users.sourceforge.net
</author>
 <class>NotchPopup</class>
 <widget class="QDialog" name="NotchPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>129</width>
    <height>184</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Notch filter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="graphic.qrc">
    <normaloff>:/notch</normaloff>:/notch</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px;</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lbFrq">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Central frequency</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sbFreq">
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox::down-button, QSpinBox::up-button {
width: 16px;
height: 16px;
} 


QSpinBox::up-arrow {
    image: url(:/icons/up);
    width: 14px;
    height: 14px;
}


QSpinBox::down-arrow {
    image: url(:/icons/down);
    width: 10px;
    height: 10px;
}</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="suffix">
      <string notr="true"> Hz</string>
     </property>
     <property name="value">
      <number>99</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbNW">
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lbHz_2">
        <property name="text">
         <string>Notch width</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbBand">
        <property name="styleSheet">
         <string notr="true">/* hide buttons to use slider */
QSpinBox::down-button, QSpinBox::up-button {
width: 1px;
height: 1px;
} </string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="suffix">
         <string notr="true"> Hz</string>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="hsBand">
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="sliderPosition">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="tbOk">
       <property name="toolTip">
        <string>update this notch</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="graphic.qrc">
         <normaloff>:/ok</normaloff>:/ok</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbDel">
       <property name="toolTip">
        <string>delete this notch</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="graphic.qrc">
         <normaloff>:/less</normaloff>:/less</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbCancel">
       <property name="toolTip">
        <string>close this popup</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="graphic.qrc">
         <normaloff>:/cancel</normaloff>:/cancel</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="graphic.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>hsBand</sender>
   <signal>valueChanged(int)</signal>
   <receiver>sbBand</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sbBand</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hsBand</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
